addons/udes_stock/models/stock_picking_type.py

```diff
@@ -334,6 +334,14 @@ class StockPickingType(models.Model):
         " for any empty picking in the system.",
     )
 
+    # Maybe `reserve_pallet_per_picking` as we don;t know about sale orders in
```

Comment:
Agree about the name.
Do we have another setting to enable matrix picking? or this is the setting that will be used to know it?

Reply:
This would be the setting.